### PR TITLE
Update note feature in user guide

### LINKED ISSUE #89: Docs: Update note command in user guide
Referencing #90, we need to update the user guide for the note command.

### PR DESCRIPTION
**Pull request by ruijietay**
Referencing #90, we need to update the `note` feature in the user guide.

Also made some minor changes to the introduction of the user guide, as well as minor changes to update `clear` command.

Closes #89.

### CODE REVIEW

**Comment by jjchee77** [comment]
Might want to change "fitness trainer" to "personal trainer" as that is our target audience and I think there's a difference between the two.

Otherwise, LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -109,6 +109,19 @@ Examples:
...
+Format: `note INDEX note/NOTE`
+
+* Edits the note of the client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
+* Existing note will be updated to the input note.
+
+> While this can also be done using the `edit` command, this `note` command serves as a faster way for users to directly modify a note.
+
+Examples:
+*  `note 1 note/History of asthma` Edits the note of the 1st client to `History of asthma`.
+*  `edit 2 note/Previously sprained both ankles` Edits the note of the 2nd client to `Previously sprained both ankles`.
```
Comment (RyanNgWH) [inline docs/UserGuide.md:123]: May be nicer to add a dash between the command and the description?

```suggestion
*  `note 1 note/History of asthma` - Edits the note of the 1st client to `History of asthma`.
*  `edit 2 note/Previously sprained both ankles` - Edits the note of the 2nd client to `Previously sprained both ankles`.
```

**Review by RyanNgWH — APPROVED** [review]
Just a small suggestion on formatting. Otherwise, looks good to merge!

File: docs/UserGuide.md (hunk 1 above)
Comment (ruijietay) [reply docs/UserGuide.md]: Good call!

**Review by jjchee77 — APPROVED** [review]
LGTM!

**Pull request merged**